src/components/molecules/PlanRoom/RoomTables/TableDNDTypes/index.tsx

```diff
@@ -0,0 +1,5 @@
+const ItemTypes = {
```

Comment:
If that type is necessary, place it inside `models` directory.

Reply:
I will move this to models